lib/console/procps_utils.pm

```diff
@@ -0,0 +1,60 @@
+# SUSE's openQA tests
+#
+# Copyright © 2021 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Basic functions for testing vmstat.
+# Maintainer: Deepthi Yadabettu Venkatachala<deepthi.venkatachala@suse.de>
+#
+package console::procps_utils;
+
+use base Exporter;
+use Exporter;
+
+use base "consoletest";
+use testapi;
+use strict;
+use warnings;
+
+our @EXPORT = qw(read_memory_cpu average);
+
+# Returns a list with two array referencesarray of amount of idle memory and CPU usage from the vmstat output log file.
+# Input :  File which contains vmstat output.
+sub read_memory_cpu {
+
+        my $file_name = shift;
+        my @res_mem = split '\n',  script_output "awk '{print \$4}' $file_name";
+        my (@mem_num, @cpu_num);
+        foreach my $elem (@res_mem) {
+                if ($elem =~ m/\d+/) {
+                        push @mem_num, $elem;
+                }
+        }
+        my @res_cpu =  split '\n',  script_output "awk '{print \$12}' $file_name";
+        foreach my $cpu_elem (@res_cpu) {
+                if ($cpu_elem =~ m/\d+/) {
+                        push @cpu_num, $cpu_elem;
+                }
+        }
+        return (\@mem_num, \@cpu_num);
+}
+
+# Returns the average of numbers.
+# Input:  Array of numbers.
```

Comment:
Please add some POD docs to methods here: https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/lib/utils.pm#L123